components/tabs/tabs-bottom.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { TabStatus, TabVarient } from 'components/utils/prop-types'
+import useTheme from '../styles/use-theme'
+
+export type BottomCpt = React.FC<{
+  status: TabStatus
+  varient: TabVarient
+}>
+
+const Bottom: BottomCpt = ({ varient, status }) => {
+  const { palette } = useTheme()
+  return varient === 'line' ? (
+    <div
+      className="bottom"
+      style={{
+        backgroundColor: palette.cTheme0,
+        opacity: status === 'active' ? '1' : '0',
+        transform: status === 'active' ? '1' : '0.75',
```

Comment:
Is it missing transform props config here?

Reply:
yes